batch/tests/test_basics.py

```diff
@@ -155,3 +162,18 @@ def test_service_account_job(job_name, service_account):
 def test_ssd_job(job_name: str, disk_name: str, capsys: "pytest.CaptureFixture[str]"):
     job = create_local_ssd_job(PROJECT, REGION, job_name, disk_name)
     _test_body(job, additional_test=lambda: _check_logs(job, capsys))
+
+
+@flaky(max_runs=3, min_passes=1)
+def test_pd_job(job_name, disk_name):
+    region = "europe-north1"
+    zone = "europe-north1-c"
+    existing_disk_name = "permanent-batch-testing"
+    job = create_with_pd_job(
+        PROJECT, region, job_name, disk_name, zone, existing_disk_name
+    )
+    _test_body(
+        job,
+        additional_test=lambda: _check_policy(job, job_name, disk_name),
+        region=region,
+    )
```

Comment:
Can we check for both disks: disk_name and existing_disk_name? And maybe don't assume their order in the disks[] list in the policy object. I'm not sure if it's guaranteed to be in any kind of order.